File: Sorting_in_Python/bubble.py
def py_bubble(arr, n):
    flag = False
    for i in range(n):
        flag = False
        for j in range(0, n-i-1):
            if arr[j] > arr[j + 1]:
                flag = True
                arr[j], arr[j + 1] = arr[j + 1], arr[j]
        if not flag:
            return

File: Sorting in java/Bubble.java
package bpack;

public class Bubble
{
public void bubbleSort(long arr[],long n)
{
    long i=0;
    long j=0;
    long temp=0;
    for(i=0;i<n-1;i++)
    {
        int flag=0;
        for(j=0;j<n-i-1;j++)
        {
            if(arr[j]>arr[j+1])
            {
                temp=arr[j];
                arr[j]=arr[j+1];
                arr[j+1]=temp;

                flag=1;
            }
            if(flag==0)
            break;
        }
    }
}
}